### ISSUE TITLE
Coloration doesn't recognize actual tabs

### ISSUE THREAD

**Issue by iSheeft** [atom/language-make#10]
In the following, clean is colored only if, instead of hard tabs, there are two spaces :
```make
all: foo.ml
    ocamlopt -o out foo.ml

clean:
    rm *.cmi *.cmo *.cmx *.o
```

**Comment by MatthewMueller** [comment]
+1

**Comment by EliaGeretto** [comment]
The problem is still present in Atom 0.187.0, the use of hard tabs compromises the highlighting of the following lines.

**Comment by jackwilsdon** [comment]
:+1: Still present in 0.196.0

**Comment by thibault-martinez** [comment]
+1

**Comment by codykrieger** [comment]
+1

**Comment by Kerosene2000** [comment]
```Makefile
rere: ffclean all

$(NAME): $(SUBMODULES) $(LIBRARIES) $(OBJS)
	@$(CC) -o $@ $(OBJS) $(BIN_FLAGS)
	@echo $@ "updated!"

clean:
	@rm -rf $(OBJS)
```

And if we use rule that use `$(VARIABLE)`, coloration is not added to next rules...

**Comment by 50Wliu** [comment]
This has been fixed and will be out in Atom 1.0.4.

**Comment by Kerosene2000** [comment]
could thank you ! :smile:
